commands.c

```diff
@@ -1068,10 +1068,12 @@ int mutt_check_traditional_pgp (HEADER *h, int *redraw)
   else
   {
     for (i = 0; i < Context->vcount; i++)
-      if (Context->hdrs[Context->v2r[i]]->tagged && 
-	  !(Context->hdrs[Context->v2r[i]]->security & PGP_TRADITIONAL_CHECKED))
-	rv = _mutt_check_traditional_pgp (Context->hdrs[Context->v2r[i]], redraw)
-	  || rv;
+      if ((Context->hdrs[Context->v2r[i]]->tagged) && 
+           (Context->hdrs[Context->v2r[i]]->security & PGP_TRADITIONAL_CHECKED))
+      {
+        rv = _mutt_check_traditional_pgp (Context->hdrs[Context->v2r[i]],redraw)
+                  || rv;
+      }
   }
```

Comment:
Clang format doesn't include extra characters, like braces. It only puts whitespace into files^^